nette/notejam/app/presenters/NotePresenter.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace App\Presenters;
+
+use App\Forms\Note\DeleteNoteFormFactory;
+use App\Forms\Note\EditNoteFormFactory;
+use App\Forms\Note\NewNoteFormFactory;
+use App\Model\NoteManager;
+use App\Model\PadManager;
+use Nette;
+use App\Model;
+use Nette\Application\BadRequestException;
+
+
+class NotePresenter extends SecuredBasePresenter
+{
+
+	/** @var PadManager @inject */
+	public $padManager;
+
+	/** @var NoteManager @inject */
+	public $noteManager;
+
+	/** @var EditNoteFormFactory @inject */
+	public $editPadFormFactory;
+
+	/** @var DeleteNoteFormFactory @inject */
+	public $deletePadFormFactory;
+
+	/** @var NewNoteFormFactory @inject */
+	public $newPadFormFactory;
+
+	/** @var int @persistent */
+	private $id;
```

Comment:
They're not, the `getParameter()` doesn't care about persistence. It works, but it's not persistent :)